NBXplorer/wwwroot/api.json

```diff
@@ -3398,6 +3467,138 @@
           }
         }
       }
+    },
+    "/cryptos/{cryptoCode}/node/rpc": {
+      "post": {
+        "summary": "Proxy RPC call to the underlying node",
+        "description": "Allows you to make RPC calls to the underlying cryptocurrency node (e.g., Bitcoin Core) via NBXplorer. This endpoint acts as a proxy, forwarding your RPC requests to the node and returning the node's responses.",
```

Comment:
Add link to https://www.jsonrpc.org/specification and https://developer.bitcoin.org/reference/rpc/